src/utilities/sequence-editor/sequencer-grammar-constants.ts

```diff
@@ -1,3 +1,29 @@
+import type { SyntaxNode } from '@lezer/common';
+
+export const TOKEN_ACTIVATE = 'Activate';
+export const TOKEN_GROUND_BLOCK = 'GroundBlock';
+export const TOKEN_GROUND_EVENT = 'GroundEvent';
+export const TOKEN_LOAD = 'Load';
 export const TOKEN_COMMAND = 'Command';
+export const TOKEN_REQUEST = 'Request';
 export const TOKEN_REPEAT_ARG = 'RepeatArg';
 export const TOKEN_ERROR = '⚠';
```

Comment:
Can these constants be moved under the `/constants` directory for consistency? And leave the function in this `directory` but maybe the file name is just `sequence-grammar.ts` or something that might make more sense.